lib/migration.js

```diff
@@ -166,6 +175,18 @@ module.exports = function(IBMDB) {
     // ------------------------------------------------------------------------
     sql = sql.concat(self.buildIndexes(model));
 
+    // add back the primary key index
+    // this snippet is required in here rather than the `buildIndexes` function
+    // because DB2 automatically creates an index with a primary key when a
```

Comment:
This statement isn't entirely accurate.  In ibmdb.js in buildColumnDefinitions we explicitly create primary keys for the ID column where we could have just defined it as a unique index and not had a primary key.  This isn't a DB2 issue that is causing us to special case things, it is our use of the DB2 SQL syntax.